sklearn/metrics/_regression.py

```diff
@@ -500,12 +507,16 @@ def mean_squared_log_error(
             "targets contain negative values."
         )
 
-    return mean_squared_error(
+    msle = mean_squared_error(
         np.log1p(y_true),
         np.log1p(y_pred),
         sample_weight=sample_weight,
         multioutput=multioutput,
     )
+    if squared:
+        return msle
+    else:
+        return np.sqrt(msle)
```

Comment:
Why not pass squared to `mean_squared_error` directly?

```suggestion
    return mean_squared_error(
        np.log1p(y_true),
        np.log1p(y_pred),
        sample_weight=sample_weight,
        multioutput=multioutput,
        squared=squared,
    )
```

Reply:
Yes, I was tired when I suggested the previous change :)

Reply:
@ogrisel  I think according to the code you proposed it will not check for the squared parameter and return the `mean_squared_error` directly. What do you think?